packages/@o3r/schematics/src/utility/wrapper.ts

```diff
@@ -47,16 +48,20 @@ For more details and instructions on how to change these settings, see https://g
 
       if (isReplyPositive) {
         const version = JSON.parse(readFileSync(path.join(__dirname, '..', '..', 'package.json'), 'utf-8')).version;
-        context.addTask(
-          new NodePackageNgAddTask(
-            '@o3r/telemetry',
-            {
-              dependencyType: NodeDependencyType.Dev,
-              version
+        const installTelemetryRule = setupDependencies({
+          dependencies: {
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            '@o3r/telemetry': {
+              inManifest: [{
+                range: `~${version}`,
+                types: [NodeDependencyType.Dev]
+              }],
+              requireInstall: true
```

Comment:
Not sure the installation is required explicitly.
This function is mainly in case of `ngAdd` chain to make sure the installation is run for this package before running the following `ngAdd` schematic. I think the installation is not really mandatory here for following `ngAdd`